airbyte-integrations/connectors/source-retently/setup.py

```diff
@@ -6,7 +6,7 @@
 from setuptools import find_packages, setup
 
 MAIN_REQUIREMENTS = [
-    "airbyte-cdk",
+    "airbyte-cdk~=0.2",
```

Comment:
Can you please explain this change?

Reply:
No direct explanation, except for trying to pin it to a specific version like most other connectors I think? I can remove the dependency pin if you prefer?